### PR TITLE
Merge development into main

### PR DESCRIPTION
**Pull request by Paldiu**
Merge development into main

### CODE REVIEW

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_mute.java [hunk 1]
```diff
@@ -130,17 +130,19 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
...
-            if (reason != null)
-            {
-                msg(player, ChatColor.RED + "Reason: " + ChatColor.YELLOW + reason);
-            }
+            
             if (quiet)
             {
                 msg("Muted " + player.getName() + " quietly");
-                return true;
+                return true;  // doesn't announce reason
```
Comment (VideoGameSmash12) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_mute.java:137]: the code you moved didn't ever announce the reason, wtf?

**Review by VideoGameSmash12 — COMMENTED** [review]
just one question

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_mute.java (hunk 1 above)
Comment (Paldiu) [reply src/main/java/me/totalfreedom/totalfreedommod/command/Command_mute.java:137]: Note the "quietly". Why would it announce the reason if it was intended for quiet announcement?

**Comment by Wild1145** [comment]
What is the purpose of this PR?

**Pull request closed without merging**